packages/react-core/src/demos/examples/Card/CardAggregateStatus.tsx

```diff
@@ -75,11 +85,19 @@ const cardData: CardData = {
       title: '10 Hosts',
       content: [
         {
-          icon: <ExclamationCircleIcon color={global_success_color_100.var} />,
+          icon: (
+            <Icon status="success">
+              <ExclamationCircleIcon />
+            </Icon>
```

Comment:
I noticed we have a color/icon mismatch or two, so I [compared to core](https://pf6.patternfly.org/components/card/html-demos#aggregate-status-card) and am wondering if we can make some changes to match core.

```suggestion
            <Icon status="success">
              <CheckCircleIcon />
            </Icon>
```